src/index.ts

```diff
@@ -275,23 +279,47 @@ class LiskMigrator extends Command {
 
 					// Ask user to manually stop Lisk Core v3 and continue
 					const isLiskCoreV3Stopped = await cli.confirm(
-						"Please stop Lisk Core v3 to continue. Type 'yes' and press Enter when ready. [yes/no]",
+						"Please stop Lisk Core v3 to continue. Type 'yes' and press Enter when ready. [yes/no] ",
 					);
 
 					if (isLiskCoreV3Stopped) {
+						let numTriesLeft = 3;
+						// eslint-disable-next-line no-plusplus
+						while (numTriesLeft--) {
```

Comment:
😅 the lint is warning already?

while (numTriesLeft) {
  
  numTriesLeft = numTriesLeft - 1;
}